Test1.gb.gocd.yaml

```diff
@@ -131,4 +145,48 @@ pipelines:
             - docker
             tasks:
             - script: |
-                echo command to run smoke test 
\ No newline at end of file
+                echo command to run smoke test 
+  Capacity_testing:
+    group: AT20Group
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environment_variables:
+      GIT_COMMIT_HASH: 1.2
+      TARGET_HOST: 192.168.0.46
+    materials:
+      Test1:
+        pipeline: Test1
+        stage: PackageAndPublish
+    stages:
+    - DeployToAUTO:
+        fetch_materials: false
+        keep_artifacts: false
+        clean_workspace: false
+        approval:
+          type: success
+          allow_only_on_success: false
+        jobs:
+          deployToAUTO:
+            timeout: 0
+            resources:
+            - docker
+            tasks:
+            - fetch:
+                pipeline: Test1
+                stage: Test
+                job: unit-test
+                source: gitHash.txt
+                is_file: true
+                run_if: passed
+            - fetch:
+                pipeline: Test1
+                stage: PackageAndPublish
+                job: docker
+                source: docker-compose.gb.yaml
+                is_file: true
+                run_if: passed
+            - script: |
+                export GIT_COMMIT_HASH=$(cat gitHash.txt)
+                export TAG_VERSION=${GIT_COMMIT_HASH} 
+                DOCKER_HOST=ssh://$TARGET_HOST docker-compose -f docker-compose.gb.yaml up -d
```

Comment:
Add stage to run capacity testing with a step: command here to run capacity testing

Reply:
I added thte stage